src/interactive/Dropdown.jsx

```diff
@@ -71,27 +113,38 @@ class Dropdown extends React.PureComponent {
 			ref => !ref.contains(e.target)
 		);
 
-		if (isNotDropdownClick) {
+		if (isNotDropdownClick || (!isNotDropdownClick && this.props.closeOnBodyClick)) {
```

Comment:
I'm assuming this is the fix (always good to call out the specific changes with a comment in PRs that have a lot of 'maintenance noise' like this).

No need for `!isNotDropdownClick` here - `if (isNotDropdownClick || this.props.closeOnBodyClick)` is equivalent. However, since `this.props.closeOnBodyClick` is like a universal override, I recommend giving it special treatment at the top of the handler

```js
onBodyClick = e => {
  if (this.props.closeOnBodyClick) {
    closeContent(e);
  }
  // continue with `isNotDropdownClick` logic
}
```

That avoids unnecessary calculations and makes the override behavior very clear

Reply:
sounds good, thanks! and yes, should've highlighted that